code/game/area/space_station_13_areas.dm

```diff
@@ -1173,6 +1173,11 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 	icon_state = "sec_cremation"
 	sound_environment = SOUND_AREA_SMALL_ENCLOSED
 
+/area/security/interrogation
```

Comment:
please maintain current functionality by adding it to `/datum/bounty/item/security/paperwork`

(also please add a comment at the start of the security entries about this.)